roles/fio-bench/tasks/main.yml

```diff
@@ -25,21 +75,59 @@
           metadata:
             labels:
               app: fio-benchmark
-          spec:
-            containers:
-            - name: benchmark-server
-              image: "quay.io/benchmark-operator/fio:latest"
-              command: ["/bin/sh", "-c"]
-              args:
-                - "cat /tmp/fio/fiojob;
-                   fio /tmp/fio/fiojob"
-              volumeMounts:
-              - name: config-volume
-                mountPath: "/tmp/fio"
-            volumes:
-            - name: config-volume
-              configMap:
-                name: fio-test
-            restartPolicy: Never
-  when: fio.clients > 0
+      spec:
+        containers:
+        - name: bench-server
+          image: "quay.io/jtaleric/fio:testing"
+          command: ["/bin/sh","-c"]
+          args:
+           - "cat /tmp/fio/fiojob;
+              fio /tmp/fio/fiojob"
+          volumeMounts:
+          - name: config-volume
+            mountPath: "/tmp/fio"
+          - name: ceph-volume
+            mountPath: "/mnt/pvc"
+        volumes:
+        - name: config-volume
+          configMap:
+            name: fio-test
+        - name: ceph-volume
+          persistentVolumeClaim:
+            claimName: claim{{item}}
+        restartPolicy: Never
+  when: fio.clients > 0 and fio.storageclass is defined
+  with_sequence: start=1 count={{fio.clients}}
+
+- name: Start Server(s) - no PVC
+  k8s:
+    definition:
+      kind: Pod
+      apiVersion: v1
+      metadata:
+        name: '{{ meta.name }}-fio-client-{{item}}-benchmark'
+        namespace: '{{ operator_namespace }}'
+      spec:
+        ttlSecondsAfterFinished: 600
+        template:
+          metadata:
+            labels:
+              app: fio-benchmark   
+      spec:
```

Comment:
I think the indentation is wrong and needs to be shifted from L116-L131 2 double spaces to right

Reply:
I am not sure.. It was working before with this indentation only.

Reply:
Correct the indentation works if the kind is pod, but the kind was updated from job to pod so thus the confusion.